synapse/models/infotech.py

```diff
@@ -1217,6 +1232,20 @@ def getModelDefs(self):
                         'doc': 'An array of ATT&CK technique IDs addressed by the mitigation.',
                     }),
                 )),
+                ('it:mitre:attack:flow', {}, (
+                    ('name', ('str', {}), {
+                        'doc': 'The name of the attack-flow diagram.'}),
+                    ('json', ('data', {'schema': attack_flow_schema_2_0_0}), {
```

Comment:
Maybe name this `data` ? Most of our `data` type properties use that as the name. :corn: :white_flower: :blue_heart:

Reply:
@invisig0th you did the model updates for this PR; change to `data` or leave as `json`?